scripts/get-job-matrix.py

```diff
@@ -143,6 +143,14 @@ class MakefileTest(TypedDict):
     "python": "3.8.x",
 }
 
+ALL_VERSION_SET = {
+    "name": ["all"],
```

Comment:
Not sure we use the name anywhere really, other than having it there for debugging when the version set is printed out, but for the other languages `name` is a string, not a list.

```suggestion
    "name": "all",
```

Reply:
This was a bit of a hack to make the loop below work, but I'll do this differently.

Reply:
Made it so that the name is calculated in the loop.  This is actually displayed in the job name, so all the names should be different anyway.  I considered including the version names here, but that's a bit of a long list and would just get more confusing I think.